modules/jdbc-test/src/test/resources/container-license-acceptance.txt

```diff
@@ -1,2 +1,3 @@
 mcr.microsoft.com/mssql/server:2017-CU12
+mcmoe/mssqldocker:latest
```

Comment:
please remove after converting into a unit test, we should not accept licenses of 3rd party images